### PR TITLE
Fixing typos for `s1_cslc_radar` and `s1_resample`

### PR DESCRIPTION
**Pull request by mfangaritav**
This PR is to add the `compute_ground_to_sat_east` and `compute_ground_to_sat_north` options in the yaml schema and to fix few typos in the s1_resample.py and utils/runconfig.py for the cslc workflow.

### CODE REVIEW

File: src/compass/schemas/s1_cslc_radar.yaml [hunk 1]
```diff
@@ -78,6 +78,8 @@ rdr2geo_options:
    compute_local_incidence_angle: bool(required=False)
    # Enable/disable azimuth angle output
    compute_azimuth_angle: bool(required=False)
+   compute_ground_to_sat_east: bool(required=False)
+   compute_ground_to_sat_north: bool(required=False)
```
Comment (scottstanie) [inline src/compass/schemas/s1_cslc_radar.yaml:82]: Could you add the corresponding entries in the `defaults` YAML as well?

Comment (mfangaritav) [reply src/compass/schemas/s1_cslc_radar.yaml:82]: No problem, I added the options in the defaults

**Review by scottstanie — APPROVED** [review]

**Comment by scottstanie** [comment]
Thanks for the patience @mfangaritav

**Comment by jhkennedy** [comment]
@scottstanie Thank you! Just noticed this went through. Any idea when we can expect the next release?

**Comment by scottstanie** [comment]
> @scottstanie Thank you! Just noticed this went through. Any idea when we can expect the next release?

I think it should happen some time in the next month- We're currently checking for any other Sentinel-1C pending changes we'll need to make to have OPERA be compatible for the new data.

**Pull request merged**